komissarova_workspace_7/src/app/reducers/usersReducer.js

```diff
@@ -0,0 +1,30 @@
+export default function reducer(state = {
+  user: {
```

Comment:
```suggestion
  user: []
```
так как `user` будет хранить массив пользовательских данных